emuu-client/src/components/UserProfile/Profile.js

```diff
@@ -16,51 +16,47 @@ import {
   onSnapshot,
 } from "firebase/firestore";
 
-
 function Profile() {
+  useEffect(() => {
+    console.log(JSON.parse(localStorage.getItem("user")));
+    getDocs(collection(db, "Users")).then((data) => {
+      console.log(data);
+    });
+  }, []);
 
-    useEffect(()=>{console.log(JSON.parse(localStorage.getItem("user")));getDocs(collection(db, "Users")).then(data =>{
-    console.log(data)
-    })}, [])
-
-
-function verifyJpeg(filename){
-const fnArr = filename.split(".");
-if(fnArr[fnArr.length-1]== "jpeg"||fnArr[fnArr.length-1]== "jpg" )return true;
-return false;
-}
-
-
-  function uploadBackground(e){
-  let file = e.target.files[0];
-
-  if(!verifyJpeg(file.name))return;
-      const storage = getStorage();
-      const storageRef = ref(storage, '/images/'+file.name);
-
-      // 'file' comes from the Blob or File API
-      uploadBytes(storageRef, file).then((snapshot) => {
-   getDownloadURL(storageRef).then(URL => console.log(URL))
-      });
+  function verifyJpeg(filename) {
+    const fnArr = filename.split(".");
+    if (fnArr[fnArr.length - 1] == "jpeg" || fnArr[fnArr.length - 1] == "jpg")
+      return true;
+    return false;
   }
 
+  function uploadBackground(e) {
+    let file = e.target.files[0];
 
-  function uploadProfile(e){
-  let file = e.target.files[0];
-  if( !verifyJpeg(file.name)) return;
-    const storage=getStorage();
-    const storageRef=ref(storage, '/images/'+file.name);
+    if (!verifyJpeg(file.name)) return;
+    const storage = getStorage();
+    const storageRef = ref(storage, "/images/" + file.name);
 
+    // 'file' comes from the Blob or File API
     uploadBytes(storageRef, file).then((snapshot) => {
-    getDownloadURL(storageRef).then(URL =>console.log(URL))
+      getDownloadURL(storageRef).then((URL) => console.log(URL));
     });
+  }
 
-}
+  function uploadProfile(e) {
+    let file = e.target.files[0];
+    if (!verifyJpeg(file.name)) return;
+    const storage = getStorage();
+    const storageRef = ref(storage, "/images/" + file.name);
 
+    uploadBytes(storageRef, file).then((snapshot) => {
+      getDownloadURL(storageRef).then((URL) => console.log(URL));
```

Comment:
What does console logging the url achieve? should be meaningful code or comment it as a place holder or something to keep not we need to update later on.